ntbs-service/Services/ValidationService.cs

```diff
@@ -92,7 +92,9 @@ private ContentResult GetValidationResult(object model, string key)
         {
             pageModel.TryValidateModel(model);
 
-            return ModelState()[key] == null ? ValidContent() : pageModel.Content(ModelState()[key].Errors[0].ErrorMessage);
+            return (ModelState()[key] == null || ModelState()[key].ValidationState == ModelValidationState.Valid)
```

Comment:
I would probably extract `ModelState()[key]` into a variable at this point :)
Perhaps it'd be worth turning the ternary operator into an if-else too, what do you think?